src/utilities/util_verify.c

```diff
@@ -13,10 +13,12 @@ usage(void)
 {
     static const char *options[] = {"-d config",
       "display underlying information during verification", "-s",
-      "verify against the specified timestamp", NULL, NULL};
+      "verify against the specified timestamp", "-u",
+      "display the application data when dumping with configuration dump_blocks or dump_pages",
+      NULL, NULL};
```

Comment:
Could you reshuffle the lines here so the option and its description sort on the same line? Or will auto-formatting undo that?

Reply:
Unfortunately s_all auto formats tihs